### ISSUE TITLE
Finalize event details page

### ISSUE THREAD

**Issue by jschen5** [kamiyo/sycpiano#74]
Notes from discussion:
- [x] remove date and time from left column
- [x] move venue name to under the title (remove address), and link it to the address in google maps (maybe with miniaturized marker icon to the left)
- [ ] make program text larger (since it's now going to be the only text below the collaborators divider)

**Comment by ayc92** [comment]
@kamiyo @jschen5 
What do you guys think about adding a thumbnail or image for each event? I think it'd really help make the details part more appealing. Also, we should also have links to the collaborators' sites if available.

Also, it'd also be great if we modify the `piece` schema to separate out "name of piece" and "composer", just so we can do some more interesting things with styling the text.

So, just to summarize, here are some extra proposals:
- [ ] Image/thumbnail per event
- [ ] Collaborator page URL per collaborator
- [ ] Split `piece` into "name" and "composer"
All these will require schema migrations.

**Comment by kamiyo** [comment]
@ayc92 @jschen5 Sure, we can try it.

I can work on the piece schema thing, but I just want to get an idea how we should populate it, because right now, I think they're just in one line, and so it's hard to parse that out.

**Comment by kamiyo** [comment]
@ayc92 on second thought, I think the collaborator URL isn't necessary (because mostly it would be with orchestras, and the link to the event would pretty much go to their page anyways). A lot of times chamber music is with people that don't have their own pages anyways. I mean it won't take much to add it in (we can just add a column, and not populate it for now), but that's my two cents on that.

**Comment by ayc92** [comment]
Yeah, re: the collaborator URL, we can shelve this as a nice-to-have, and focus on the thumbnail and splitting the `piece` model up.

**Comment by kamiyo** [comment]
For the model, I was thinking maybe we have a TBD flag somewhere - that way if pieces are blank, then

**Comment by kamiyo** [comment]
nvm on the TBD

**Comment by kamiyo** [comment]
@ayc92 @jschen5 What do you think about using this to scrape open graph tags for the thumbnail picture?: https://github.com/jshemas/openGraphScraper

Since we already have some urls we can try that. We also can maybe do something else programmatic like if there isn't opengraph thumbnail tags: have a local db of composers and their pictures, and then just loop through the composers of the pieces of the current event until it finds one that we have a picture of. If all that fails, then we can just have it blank. Let me know your thoughts.

**Comment by ayc92** [comment]
@kamiyo That sounds like a good idea :)

**Comment by ayc92** [comment]
Closing due to https://github.com/kamiyo/sycpiano/pull/83
